lib/asciidoctor/reader.rb

```diff
@@ -835,10 +835,9 @@ def preprocess_include raw_target, raw_attributes
         warn %(asciidoctor: ERROR: #{line_info}: maximum include depth of #{@maxdepth[:rel]} exceeded)
         return false
       end
-      if ::RUBY_ENGINE_OPAL
+      if ::RUBY_ENGINE_OPAL && @document.attributes.has_key? 'env-browser'
```

Comment:
@mojavelinux outdated diff because I forgot to add parenthesis but the comment is still relevant and I will amend my commits once we have agreed